### PR TITLE
Show <remarks> documentation in Quick Info

### PR DESCRIPTION
**Pull request by sharwell**
Builds on #42128 

Originally I was more concerned that this would clutter the Quick Info presentation. However, I found that the core library does not include any `<remarks>` documentation (at least in **mscorlib.xml**). The presentation used in this pull request has `<remarks>` rendered exactly the same way as additional `<para>` elements within a `<summary>` section.

image

### CODE REVIEW

**Comment by sharwell** [comment]
Here's a screenshot from a property with longer remarks. I cropped the right side because I'm on a very wide screen and it didn't wrap.

image

**Comment by miloush** [comment]
I wonder if the remarks section in the tooltip could be somehow indicated, either visually (font style/separator/indentation etc.), or maybe preferably just by saying "Remarks:".

**Comment by sharwell** [comment]
@miloush For the examples I found, it seemed to hurt more than help. The paragraph separation seems to work well for these two cases:

1. Users who treat the `<summary>` as either the first sentence of, or the "tl;dr" of the `<remarks>`
1. Users who write longer `<summary>` text, but leave out `<remarks>` because it was never visible to anyone before

**Comment by tamlin-mike** [comment]
@sharwell
> Users who write longer `<summary>` text, but leave out `<remarks>` because it was never visible to anyone before

Just wanted to chime in that this is the exact reason I have almost never used the `remarks` tag, even that I have really _wanted_ to - the IDE has <strike>worthless</strike> lousy support for displaying documentation in pop-ups. Compare this with e.g. IntelliJ IDEA to see how I (and most likely the vast majority of developers) would have liked this to work.

TL;DR; Thanks for fixing up old crap.

**Comment by sharwell** [comment]
Design review conclusion: This feature seems likely to work exactly as expected for users who want to use `<remarks>` elements, and doesn't show for other users.

**Comment by vatsalyaagrawal** [comment]
Design meeting notes:
- Approved for merge.
